### PR TITLE
Silence warnings about unknown target from templates when the target …

### PR DESCRIPTION
**Pull request by mpreisler**
…is actually supported

Don't spit out unknown target warnings when the template just doesn't support it.

### CODE REVIEW

File: shared/templates/template_common.py [hunk 1]
```diff
@@ -206,7 +207,12 @@ def csv_map(self, filename, language):
 
             except UnknownTargetError as e:
                 if self.action == ActionType.BUILD:
-                    sys.stderr.write(str(e) + "\n")
+                    # If lang is one of these it just means that the template
+                    # doesn't implement that language. It doesn't mean that the
+                    # target is invalid.
+                    if e.lang not in ["oval", "ansible", "bash", "puppet",
```
Comment (cipherboy) [inline shared/templates/template_common.py]: Are we likely to add new build targets sometime soon? It looks like these are hard coded in a bunch of places; you might want to document somewhere that this is another location that'll need to be updated if a new target is added.

Comment (mpreisler) [reply shared/templates/template_common.py]: yeah, should probably be somewhere in ssgcommon.py. I will look into that.

Comment (mpreisler) [reply shared/templates/template_common.py]: It's used in one other spot. I have put the list into a variable in template_common, it seems to fit well there. It's reused in generate-from-templates.py.

**Comment by dahaic** [comment]
Build logs are much nicer now, thanks @mpreisler !

**Pull request merged**